packages/demand-bloc-track-trip/src/tripTransformer.ts

```diff
@@ -84,6 +86,7 @@ export const tripTransformer = (trip: OriginalTripFollowResponse | BookATripResp
     id,
     passengers,
     meeting_point,
+    service_level_agreements = null,
```

Comment:
I would propose to move null case into final object which is returned from this function to have common code appearance)